src/data/faq.json

```diff
@@ -996,7 +996,7 @@
                                 "active": false,
                                 "textblock": [
                                     "If you have stored a valid vaccination certificate or a certificate of recovery and a valid test certificate in the app, these will also be shown in a combined display from version 2.16 onwards. You can then use the switch to toggle between the QR code of the '2G certificate' (vaccination certificate or certificate of recovery) and that of the test certificate to quickly and easily provide 2G+ proof. Both QR codes must be checked on site using the CovPassCheck app together with presentation of the ID card if 2G+ proof is required. For privacy reasons, when the certificates are checked by the CovPassCheck app, it is not apparent whether the individual has a vaccination certificate or a certificate of recovery. When scanning the QR codes, the checkers only see whether the certificate is valid or not.",
-                                    "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certificate or a certificate of recovery and a negative test?</a></li></ul>"
+                                    "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my <a href='#glossary_booster_vaccination' target='_blank'>booster vaccination</a> be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certificate or a certificate of recovery and a negative test?</a></li></ul>"

```

Comment:
This is an illegal nested link. It should be restored to the original without the nested glossary link.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!